### PR TITLE
Update supported platforms doc for iOS/macOS

### PR DESCRIPTION
**Pull request by tcobbs-bentley**

### CODE REVIEW

File: docs/learning/SupportedPlatforms.md [hunk 1]
```diff
@@ -31,8 +31,8 @@ The following Node.js versions are officially supported by the iTwin.js backend
 | ---------------- | ------------- | ------------------------------------------------------------------------------------------------------------------ | --------------- |
 | Linux            | all           | GLIBC >= 2.31, GLIBCXX >= 3.4.28                                                                                   |                 |
 | Windows          | all           | [Visual Studio 2017 C Runtime](https://support.microsoft.com/help/2977003/the-latest-supported-visual-c-downloads) |                 |
-| macOS            | x64           | >= 10.15                                                                                                           |                 |
-| macOS            | arm64         | >= 11                                                                                                              | >= iTwin.js 3.3 |
+| macOS            | x64           | **Not Supported**                                                                                                  |                 |
```
Comment (tcobbs-bentley) [inline docs/learning/SupportedPlatforms.md:34]: @aruniverse: I'm not sure if this is the best solution, but I think removing the line entirely might be confusing. I'm open to suggestions if you think something else would be better.

Comment (aruniverse) [reply docs/learning/SupportedPlatforms.md:34]: maybe dont delete the line, but say its discontinued?

Comment (aruniverse) [reply docs/learning/SupportedPlatforms.md]: might be good to mention the version we discontinued support for it?

Comment (tcobbs-bentley) [reply docs/learning/SupportedPlatforms.md]: Done.

**Pull request merged**